app/services/actions/foreman/puppet_class/import.rb

```diff
@@ -0,0 +1,30 @@
+module Actions
+  module Foreman
+    module PuppetClass
+      class Import < Actions::EntryAction
+        def resource_locks
+          :import_puppetclasses
+        end
+
+        def run
+          output[:changed] = input[:changed]
```

Comment:
If the reason was to show this in task details, defining `humanized_input` method is better way to do this